File: python/highest_lowest.py
#Solution for "Highest and Lowest" 7 kyu
#https://www.codewars.com/kata/554b4ac871d6813a03000035

def high_and_low(numbers):
    s = sorted([int(n) for n in numbers.split(" ")])
    return " ".join([str(s[-1]), str(s[0])])

File: javascript/highestLowest.js
//Solution for "Highest and Lowest" 7 kyu
//https://www.codewars.com/kata/554b4ac871d6813a03000035

function highAndLow(numbers){
  var sorted = numbers.split(' ').sort(function(a, b){return a-b})
  return [sorted[sorted.length - 1], sorted[0]].join(' ');
}
